### PR TITLE
docs: add docs for the InfluxDB outputstream

### PR DESCRIPTION
**Pull request by tim-golioth**

### CODE REVIEW

**Comment by dylanswartz** [comment]
Heads up @ChrisGammell @szczys we want to hold off merging these until we remove the feature flag in production hiding this feature.

**Review by dylanswartz — COMMENTED** [review]
Great work!

**Comment by szczys** [comment]
@dylanswartz okay, it would be good to set it as a draft PR until then.

**Comment by tim-golioth** [comment]
converted this a draft PR for the time being.

**Review by szczys — CHANGES_REQUESTED** [review]
In addition to the inline comments I left, please wrap lines at 80 columns where possible. Thanks!

File: docs/cloud/2-output-streams/8-influx/README.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+![Complete create outputstream form](./assets/create-outputstream-form.png)
+
+## Example Data
+
+Golioth Output Streams use the [Cloud Events](https://cloudevents.io) format. Because
+InfluxDB is a timeseries database, we transform events from the CloudEvents format to the
+InfluxDB [line protocol](https://docs.influxdata.com/influxdb/cloud-serverless/reference/syntax/line-protocol/) format.
+
+Here is an example of a CloudEvent on the Golioth platform:
+```
```
[the remark this answers is not in the document]
Comment (tim-golioth) [reply docs/cloud/2-output-streams/8-influx/README.md:109]: this is json, not yaml.

File: docs/cloud/2-output-streams/8-influx/README.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+    "integration_id": "644aa93df9021bf366e0a255",
+    "data": {
+      "hello": "world"
+    }
+  }
+```
+
+We transform the above CloudEvent to the following line protocol:
+
+```
```
[the remark this answers is not in the document]
Comment (tim-golioth) [reply docs/cloud/2-output-streams/8-influx/README.md:133]: this is the InfluxDB line protocol, so it's neither yaml nor json.

File: docs/cloud/2-output-streams/8-influx/assets/2023-07-20_15-37.png
[the remark this answers is not in the document]
Comment (tim-golioth) [reply docs/cloud/2-output-streams/8-influx/assets/2023-07-20_15-37.png:1]: this file isn't even used. removed.

**Comment by tim-golioth** [comment]
> > This and any other style requirements should be documented somewhere, if they aren't already.
> 
> Good idea! I [added an .editorconfig file for this setting](https://github.com/golioth/docs/pull/271) when I originally left that comment, and have just opened another PR to [add styling info to the README](https://github.com/golioth/docs/pull/272).

TIL! i didn't know neovim supports [editorconfig](http://neovim.io/doc/user/editorconfig.html).

**Comment by tim-golioth** [comment]
@szczys i made a couple minor cleanup changes after you approved. would you please take another look when it's convenient?

**Comment by szczys** [comment]
> @szczys i made a couple minor cleanup changes after you approved. would you please take another look when it's convenient?

Still LGTM!

File: docs/cloud/2-output-streams/8-influx/README.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
+---
+id: overview
+title: InfluxDB Overview
+slug: /cloud/output-streams/influx
```
Comment (dylanswartz) [inline docs/cloud/2-output-streams/8-influx/README.md:4]:
```suggestion
slug: /cloud/output-streams/influxdb
```

**Review by dylanswartz — CHANGES_REQUESTED** [review]

**Review by dylanswartz — APPROVED** [review]

**Pull request merged**